ee-feature-pack/channel/src/main/resources/wildfly-ee-channel.yaml

```diff
@@ -0,0 +1,9 @@
+---
+schemaVersion: "2.0.0"
+repositories:
+  - id: "central"
+    url: "https://repo1.maven.org/maven2/"
+  - id: "jboss-public"
+    url: "https://repository.jboss.org/nexus/content/groups/public/"
+manifest:
+  maven: "org.wildfly:${project.artifactId}:${project.version}"
```

Comment:
the maven coordinates should be broken up like:
```
manifest:
  maven:
    groupId: "org.wildfly"
    artifactId: "${project.artifactId}"
    version: "${project.version}"
```

Also does the channel need to define the manifest version? I think it would make more sense to have the major version included into the artifactId of the manifest so the updates can be consumed if needed